launch_testing_ros/test/examples/wait_for_topic_launch_test.py

```diff
@@ -70,6 +77,9 @@ def test_topics_successful(self, count):
             assert wait_for_node_object_2.wait()
             assert wait_for_node_object_2.topics_received() == expected_topics
             assert wait_for_node_object_2.topics_not_received() == set()
+            for topic_name, _ in topic_list:
+                message = wait_for_node_object_2.received_messages(topic_name).pop().data
```

Comment:
Lets check the buffer length here once you resolve the callback is only triggered once issue.

Reply:
Fixed in commit 7c9e181fb3c4a3257f5315bfe217a4748019f0c3

Reply:
Added a check in commit 8249e3440fde18ebef4c8d1403c76ea0fc01f7da